packages/nouns-webapp/src/wrappers/nounsData.ts

```diff
@@ -93,8 +92,10 @@ const filterSigners = (
         ?.nounsRepresented.length || 0;
     // don't count votes from signers who have active or pending proposals
     // but include them in the list of signers to display with a note that they have an active proposal
-    const parentProposalIsUpdatable = proposalIdToUpdate && updatableProposalIds?.includes(proposalIdToUpdate ?? 0);
```

Comment:
@eladmallel would be nice to avoid this formatting diff if possible

Reply:
generally agree, but this is what is happening with our prettier config that we recently made sure was the same; had to change this file for a rename thing, and I have auto format on save. doesn't feel important enough to move slower and turn off auto formatting for these files.

wdyt?

Reply:
@eladmallel I would just do the edit in a text editor for the rename. but not a big deal either way